cloud/aws/modules/pgadmin/main.tf

```diff
@@ -0,0 +1,165 @@
+# Allow load balancer to accept traffic on new port.
+resource "aws_security_group_rule" "pgadmin_lb" {
+  description       = "Allow tcp traffic on port 4433"
+  security_group_id = var.lb_access_sg_id
+  cidr_blocks       = ["0.0.0.0/0"] # Allow traffic from all IPs.
+  type              = "ingress"
+  from_port         = 4433
+  to_port           = 4433
+  protocol          = "tcp"
+}
+
+# Add new listener to existing load balancer for the main civiform server.
+#
+# We do this to reuse the exiting TLS infra so that we do not need to manage a separate certificate.
+resource "aws_lb_listener" "pgadmin" {
+  load_balancer_arn = var.lb_arn
+  port              = 4433
+  protocol          = "HTTPS"
+  certificate_arn   = var.lb_ssl_cert_arn
+  ssl_policy        = "ELBSecurityPolicy-2016-08" # Default policy.
+  default_action {
+    type             = "forward"
+    target_group_arn = aws_lb_target_group.pgadmin.arn
+  }
+}
+
+# Traffic from load balancer is forwarded to IPs in this target group.
+resource "aws_lb_target_group" "pgadmin" {
+  name        = "${var.app_prefix}-pgadmin"
+  port        = 80
+  protocol    = "HTTP"
+  target_type = "ip"
+  vpc_id      = var.vpc_id
+
+  # Specific config for pgadmin container.
+  health_check {
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    interval            = 20
+    timeout             = 10
+    path                = "/misc/ping"
+  }
+}
+
+# Security group for pgadmin tasks to run in.
+resource "aws_security_group" "pgadmin_tasks" {
+  name        = "${var.app_prefix}-pgadmin-tasks"
+  description = "Allow HTTP traffic on port 80."
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description     = "HTTP from load balancer"
+    from_port       = 80
+    to_port         = 80
+    protocol        = "tcp"
+    security_groups = [var.lb_access_sg_id]
+  }
+
+  egress {
+    description = "HTTPS egress"
+    from_port   = 443
+    to_port     = 443
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+  egress {
+    description     = "postgres egress"
+    from_port       = 5432
+    to_port         = 5432
+    protocol        = "tcp"
+    security_groups = [var.db_sg_id]
+  }
+}
+
+# Run a pgadmin container via a ecs service.
+resource "aws_ecs_service" "pgadmin" {
+  name            = "${var.app_prefix}-pgadmin"
```

Comment:
Can we use local.name_prefix here as well for consistency? Or you intentionally didn't use "civiform" prefix here?

Reply:
Done.